apps/web/src/components/Composer/Actions/Attachment.tsx

```diff
@@ -13,6 +11,7 @@ import {
   MediaAudioMimeType,
   MediaImageMimeType
 } from '@lens-protocol/metadata';
+import * as DropdownMenuPrimitive from '@radix-ui/react-dropdown-menu';
```

Comment:
The alias name is huge imo

Reply:
Will change it from DropMenuPrimitive to DropdownMenu